src/Message/AbstractRequest.php

```diff
@@ -30,26 +28,15 @@ public function getTransactionId()
 
     protected function sendRequest($method, $endpoint, $data = null)
     {
-        $this->httpClient->getEventDispatcher()->addListener('request.error', function (Event $event) {
-            /**
-             * @var \Guzzle\Http\Message\Response $response
-             */
-            $response = $event['response'];
-
-            if ($response->isError()) {
-                $event->stopPropagation();
-            }
-        });
-
         $httpRequest = $this->httpClient->createRequest(
             $method,
             $this->endpoint . $endpoint,
             array(
                 'Authorization' => 'Bearer ' . $this->getApiKey()
             ),
-            $data
+            json_encode($data)
```

Comment:
Okay, but it was converted automatically before? Wanted to do bring back this functionality for compatibility reasons.